lib/ansible/modules/cloud/rubrik/rubrik_cluster_version.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
+# (c) 2018 Rubrik, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+module: rubrik_cluster_version
+short_description: Retrieves the software version of the Rubrik cluster.
+description:
+    - Retrieves the software version of the Rubrik cluster.
+version_added: '2.8'
+author: Rubrik Build Team (@drew-russell) <build@rubrik.com>
+
+
+extends_documentation_fragment:
+    - rubrik_cdm
+requirements: [rubrik_cdm]
+'''
+
+EXAMPLES = '''
```

Comment:
Please can you include `- name: `'s for each example task

Reply:
Added `-name` to the example.